core-bundle/src/Twig/Runtime/InsertTagRuntime.php

```diff
@@ -12,34 +12,34 @@
 
 namespace Contao\CoreBundle\Twig\Runtime;
 
-use Contao\CoreBundle\Twig\Interop\ChunkedText;
-use Contao\InsertTags;
+use Contao\CoreBundle\InsertTag\ChunkedText;
+use Contao\CoreBundle\InsertTag\InsertTagParser;
 use Twig\Extension\RuntimeExtensionInterface;
 
 final class InsertTagRuntime implements RuntimeExtensionInterface
 {
-    private InsertTags $insertTags;
+    private InsertTagParser $insertTagParser;
 
     /**
      * @internal
      */
-    public function __construct(InsertTags $insertTags = null)
+    public function __construct(InsertTagParser $insertTagParser)
     {
-        $this->insertTags = $insertTags ?? new InsertTags();
+        $this->insertTagParser = $insertTagParser;
     }
 
     public function renderInsertTag(string $insertTag): string
     {
-        return $this->replaceInsertTags('{{'.$insertTag.'}}');
+        return $this->insertTagParser->render($insertTag);
     }
 
     public function replaceInsertTags(string $text): string
```

Comment:
Should we rename the methods here as well to match those of the `InsertTagParser`?

Reply:
I think the current names are OK, but It’s your call. I like that `raw` is included in the `replaceInsertTagsChunkedRaw()` method name.